src/solver/modeler/main.cpp

```diff
@@ -90,7 +90,7 @@ int main(int argc, const char** argv)
     std::filesystem::path studyPath(argv[1]);
     logs.info() << "Study path: " << studyPath;
 
-    if (!std::filesystem::is_directory(studyPath))
+    if (!is_directory(studyPath))
```

Comment:
While this works thanks to ADL, I think it's better to keep the namespaces for clarity & safety.
If later we added another `NS::is_directory(const std::filesystem::path&)` function, the compiler might resolve to a different function and this kind of error is very hard to detect.